docs/metrics.md

```diff
@@ -1,69 +1,112 @@
 # Fluo Metrics
 
-Fluo core is instrumented using [dropwizard metrics][1]. This allows fluo users to easily gather
-information about Fluo by configuring different reporters. While dropwizard can be configured to
-report Fluo metrics to many different tools, below are some tools that have been used with Fluo.
+A Fluo application can be configured (in [fluo.properties]) to report metrics. When metrics are 
+configured, Fluo will report will some 'default' metrics about an application that helps users 
+monitor its performance. Users can also write code to report 'application' metrics from their
+applications. Both 'application' and 'default' metrics share the same reporter configured by 
+[fluo.properties] and are described in detail below.
 
-1.  [Grafana/InfluxDB][3] - Fluo has [documentation][3] for sending metrics to InfluxDB and viewing
-    them in Grafana.
+## Configuring reporters
 
-2.  JMX - Fluo can be configured to reports metrics via JMX which can be viewed in jconsole or
-    jvisualvm.
+Fluo metrics are not published by default. To publish metrics, configure a reporter in the 'metrics'
+section of [fluo.properties]. There are several different reporter types (i.e Console, CSV, 
+Graphite, JMX, SLF4J) that are implemented using [Dropwizard]. The choice of which reporter to use
+depends on the visualization tool used. If you are not currently using a visualization tool, there
+is [documentation][grafana] for reporting Fluo metrics to Grafana/InfluxDB.
 
-3.  CSV - Fluo can be configured to output metrics as CSV to a specified directory.
+## Metrics names
 
-## Configuring Reporters
+When Fluo metrics are reported, they are published using a naming scheme that encodes additional
+information.
 
-In order to configure metrics reporters, look at the metrics section in an applications
-`fluo.properties` file. This sections has a lot of commented out options for configuring reporters.
+Default metrics start with `fluo.class` or `fluo.system` and have following naming schemes:
 
-    fluo.metrics.reporter.console.enable=false
-    fluo.metrics.reporter.console.frequency=30
+        fluo.class.APPLICATION.REPORTER_ID.METRIC.CLASS
```

Comment:
i tried adding greater/less than but i think it makes it harder to read.  i made it more explicit that all caps are variables in 7e0c08fec69e